console-framework-client/src/main/java/io/axoniq/console/framework/messaging/HandlerMetricsRegistry.kt

```diff
@@ -121,12 +120,7 @@ class HandlerMetricsRegistry(
     }
 
     private fun createTimer(handler: Any, name: String): Timer {
```

Comment:
It seems to me that this definition (`createTimer(handler: Any, name: String)`...)is redundant since it is used only once in the code.